modules/expo-bluesky-video-player/src/VideoPlayer.web.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+
+import {VideoPlayerViewProps} from './VideoPlayer.types'
+
+export class VideoPlayer extends React.PureComponent<VideoPlayerViewProps> {
+  private readonly videoPlayerRef: React.RefObject<HTMLMediaElement> =
+    React.createRef()
+
+  constructor(props: VideoPlayerViewProps | Readonly<VideoPlayerViewProps>) {
+    super(props)
+  }
+
+  static async prefetchAsync(_: string): Promise<void> {
+    console.warn('prefetchAsync is not supported on web')
+  }
+
+  private firePlayerStateChangeEvent = (isPlaying: boolean) => {
+    this.props.onPlayerStateChange?.({
+      nativeEvent: {
+        isPlaying,
+      },
+    })
+  }
+
+  async playAsync(): Promise<void> {
+    this.videoPlayerRef.current.play()
+    this.firePlayerStateChangeEvent(true)
+  }
+
+  async pauseAsync(): Promise<void> {
+    this.videoPlayerRef.current.pause()
+    this.firePlayerStateChangeEvent(false)
+  }
+
+  async toggleAsync(): Promise<void> {
+    if (this.videoPlayerRef.current.paused) {
+      await this.playAsync()
+    } else {
+      await this.pauseAsync()
+    }
+  }
```

Comment:
even though none of these are actually async on web, let's just keep the same types as what we have on native.